app/api/ada/lib/cardanoCrypto/cryptoToModel.js

```diff
@@ -22,7 +24,29 @@ export function toAdaWallet(walletInitData : AdaWalletInitData): AdaWallet {
       cwName,
       cwUnit
     },
-    cwPassphraseLU: moment().format()
+    cwType: AdaWalletTypeOption.WEB_WALLET,
+    cwPassphraseLU: moment().format(),
+  };
+}
+
+/** Convert uesr-inputted data during wallet creation to internal wallet representation */
+/** HARDWARE_WALLET */
```

Comment:
```suggestion
```

Reply:
should i remove this like as well?
https://github.com/Emurgo/yoroi-frontend/blob/4d1db2cd4983f5f006c400181d7eafea14414586/app/api/ada/lib/cardanoCrypto/cryptoToModel.js#L13

Reply:
Maybe removing this line is not the best option, but I noticed in this file you have 
`/** HARDWARE_WALLET */`
and
`/** WEB_WALLET */`
but they put in places that stop the function's comment block of appearing when you mouse over the function

Reply:
Yeah, will update with comment which makes more sense and does not break commenting convention